### PR TITLE
chore(ui): remove tech preview label for Scanner V4

### PR DESCRIPTION
**Pull request by RTann**
### Description

Move Scanner V4 into GA by removing UI labels. Somewhat reverts https://github.com/stackrox/stackrox/pull/10519

See 

### User-facing documentation

- [x] CHANGELOG is updated
- [x] Documentation PR is created and linked above - not quite, but there is a ticket which is assigned

### Testing

- [x] inspected CI results

#### Automated testing

- [x] contributed **no automated tests**

#### How I validated my change

TODO

### CODE REVIEW

File: ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsListPage.tsx [hunk 1]
```diff
@@ -70,7 +69,8 @@ function IntegrationsListPage({
     const isScannerV4 = getIsScannerV4(source, type);
     const isCloudSource = getIsCloudSource(source);
 
-    const isTechPreview = isFeatureFlagEnabled('ROX_SCANNER_V4') && type === 'scannerv4';
+    // There is currently nothing relevant in Tech Preview.
+    const isTechPreview = false;
```
Comment (sachaudh) [inline ui/apps/platform/src/Containers/Integrations/IntegrationsListPage/IntegrationsListPage.tsx:73]: Same here. You can remove the `isTechPreview` variable and the sections using it

**Review by sachaudh — COMMENTED** [review]
Thanks for taking this on!

File: ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx [hunk 2]
```diff
@@ -27,8 +27,8 @@ function ScannerV4IntegrationBanner() {
 
     const brandedText =
         branding.type === 'RHACS_BRANDING'
-            ? 'New Scanner V4 now available as Technical Preview in RHACS 4.4.'
-            : 'New Scanner V4 now available as Technical Preview in StackRox 4.4.';
+            ? 'New Scanner V4 now generally available in RHACS 4.4.'
+            : 'New Scanner V4 now generally available in StackRox 4.4.';
```
Comment (RTann) [inline ui/apps/platform/src/Components/ScannerV4IntegrationBanner.tsx:31]:
```suggestion
            ? 'New Scanner V4 now generally available in RHACS 4.5.'
            : 'New Scanner V4 now generally available in StackRox 4.5.';
```

**Review by pedrottimark — APPROVED** [review]
Neutral on pro and con of additional deletions.

For safety, if we do it, wait until we delete `ROX_SCANNER_V4` feature flag in ui code.

**Comment by sachaudh** [comment]
> Neutral on pro and con of additional deletions.
> 
> For safety, if we do it, wait until we delete `ROX_SCANNER_V4` feature flag in ui code.

That's valid. Let's keep it as is then. Since @RTann is out, I'll merge this on his behalf

File: ui/apps/platform/src/Containers/Integrations/IntegrationPage.tsx [hunk 3]
```diff
@@ -35,13 +35,13 @@ export type IntegrationPageProps = {
...
     const permissions = useIntegrationPermissions();
-    const { isFeatureFlagEnabled } = useFeatureFlags();
     const {
         pageState,
         params: { source, type, id },
     } = usePageState();
     const typeLabel = getIntegrationLabel(source, type);
-    const isTechPreview = isFeatureFlagEnabled('ROX_SCANNER_V4') && type === 'scannerv4';
+    // There is currently nothing relevant in Tech Preview.
+    const isTechPreview = false;
```
[the remark this answers is not in the document]
Comment (sachaudh) [reply ui/apps/platform/src/Containers/Integrations/IntegrationPage.tsx:44]: Let's keep it as is. We can remove those when we remove the feature flag.

**Pull request merged**